doc/releases/changelog-0.23.0.md

```diff
@@ -398,6 +398,9 @@
 
 <h3>Breaking changes</h3>
 
+* The `qml.finite_diff` function has been deleted. Please use `qml.gradients.finite_diff` to compute
+  the gradient of tapes of QNodes. Otherwise, manual implementation is required.
```

Comment:
```suggestion
  the gradient of tapes of QNodes. Otherwise, manual implementation is required.
  [(#2464)](https://github.com/PennyLaneAI/pennylane/pull/2464)
```